src/frontend/Komponenter/Behandling/Aktivitet/Aktivitet/Aksjeselskap.tsx

```diff
@@ -13,8 +17,12 @@ const Aksjeselskap: FC<{ aksjeselskap: IAksjeselskap }> = ({ aksjeselskap }) =>
             </Element>
             <Normaltekst className={'førsteDataKolonne'}>Aksjeselskap</Normaltekst>
             <Normaltekst> {aksjeselskap.navn}</Normaltekst>
-            <Normaltekst className={'førsteDataKolonne'}>Stillingsprosent</Normaltekst>
-            <Normaltekst> {aksjeselskap.arbeidsmengde + ' %'}</Normaltekst>
+            {stønadstype === Stønadstype.OVERGANGSSTØNAD && (
```

Comment:
Hadde det vært tilstrekkelig å kun sjekke for om arbeidsmengde finnes her?

Reply:
Kanskje, men vet ikke helt om jeg liker dét bedre enn å sjekke hvilken stønad verdien gjelder.. 🤔